src/group_theory/nilpotent.lean

```diff
@@ -395,3 +397,16 @@ begin
     (le_trans ((map_eq_bot_iff _).mp _) hf1)⟩,
   exact eq_bot_iff.mpr (hn ▸ (lower_central_series.map f n)),
 end
+
+lemma derived_le_lower_central (n : ℕ) : derived_series G n ≤ lower_central_series G n :=
+by { induction n with i ih, { simp }, { apply general_commutator_mono ih, simp } }
+
+theorem is_nilpotent.is_solvable (h : is_nilpotent G): is_solvable G :=
```

Comment:
I think this should be an instance
```suggestion
@[priority 100]
instance is_nilpotent.is_solvable [is_nilpotent G] : is_solvable G :=
```

Reply:
For this instance to work you will also need to make the `is_nilpotent G` a typeclass argument. In other words, you'll have to use square brackets `[` and `]` like I did above.

Reply:
Oh, sorry, of course

Reply:
If it's an instance, he correct name is `is_nilpotent.to_is_solvable`.